### ISSUE TITLE
Sketch to production

### ISSUE THREAD

**Issue by sorokinvj** [JoinColony/colonyHackathon#30]
### Colony Hackathon Submission

**Project Title**

Sketch to production
---

**Project Description**

It's just first idea that came to my mind so just for a starter – why not build an uber-like thing for putting Sketch prototypes to live? A designer upload his design, push the button and the army of small ants develop his design live app in hours. 

**Project Repository**

Github, I guess.

**Team Members and Contact info**

sorokinvj@gmail.com
Telegram @vladislavsorokin

Sure!

**Comment by mibzman** [comment]
this was pretty much what I had in mind with #28!  Want to team up?

**Comment by collinvine** [comment]
Yeah this is a cool idea. I wonder if you could add a funding layer using either the sketch or figma API

https://www.figma.com/developers
https://developer.sketchapp.com/
